ext/web/geometry.js

```diff
@@ -0,0 +1,196 @@
+
+const DOMRectPrototype = DOMRect.prototype;
+const DOMRectReadOnlyPrototype = DOMRectReadOnly.prototype;
+ObjectDefineProperty(
+  DOMRectReadOnlyPrototype,
+  SymbolFor("Deno.privateCustomInspect"),
+  {
+    __proto__: null,
+    value: function customInspect(inspect, inspectOptions) {
+      return inspect(
+        createFilteredInspectProxy({
+          object: this,
+          evaluate: ObjectPrototypeIsPrototypeOf(
+            DOMRectReadOnlyPrototype,
+            this,
+          ),
+          keys: ["x", "y", "width", "height", "top", "right", "bottom", "left"],
+        }),
+        inspectOptions,
+      );
+    },
+    enumerable: false,
+    writable: true,
+    configurable: true,
+  },
+);
+webidl.configureInterface(DOMRect);
+webidl.configureInterface(DOMRectReadOnly);
+
+const DOMQuadPrototype = DOMQuad.prototype;
+ObjectDefineProperty(DOMQuadPrototype, SymbolFor("Deno.privateCustomInspect"), {
+  __proto__: null,
+  value: function customInspect(inspect, inspectOptions) {
+    return inspect(
+      createFilteredInspectProxy({
+        object: this,
+        evaluate: ObjectPrototypeIsPrototypeOf(DOMQuadPrototype, this),
+        keys: ["p1", "p2", "p3", "p4"],
+      }),
+      inspectOptions,
+    );
+  },
+  enumerable: false,
+  writable: true,
+  configurable: true,
+});
+webidl.configureInterface(DOMQuad);
+
+const DOMMatrixPrototype = DOMMatrix.prototype;
+const DOMMatrixReadOnlyPrototype = DOMMatrixReadOnly.prototype;
+ObjectDefineProperty(
+  DOMMatrixReadOnlyPrototype,
+  SymbolFor("Deno.privateCustomInspect"),
+  {
+    __proto__: null,
+    value: function customInspect(inspect, inspectOptions) {
+      return inspect(
+        createFilteredInspectProxy({
+          object: this,
+          evaluate: ObjectPrototypeIsPrototypeOf(
+            DOMMatrixReadOnlyPrototype,
+            this,
+          ),
+          keys: [
+            "a",
+            "b",
+            "c",
+            "d",
+            "e",
+            "f",
+            "m11",
+            "m12",
+            "m13",
+            "m14",
+            "m21",
+            "m22",
+            "m23",
+            "m24",
+            "m31",
+            "m32",
+            "m33",
+            "m34",
+            "m41",
+            "m42",
+            "m43",
+            "m44",
+            "is2D",
+            "isIdentity",
+          ],
+        }),
+        inspectOptions,
+      );
+    },
+    enumerable: false,
+    writable: true,
+    configurable: true,
+  },
+);
+
+if (op_geometry_get_enable_window_features()) {
+  // https://drafts.fxtf.org/geometry/#dommatrixreadonly-stringification-behavior
+  ObjectDefineProperty(DOMMatrixReadOnlyPrototype, "toString", {
+    __proto__: null,
+    value: function toString() {
+      return op_geometry_matrix_to_string(this);
+    },
+    writable: true,
+    enumerable: true,
+    configurable: true,
+  });
+
+  // https://drafts.fxtf.org/geometry/#dom-dommatrix-setmatrixvalue
```

Comment:
This page was moved to <https://drafts.csswg.org/geometry/> earlier this week. There are no normative changes, but these links need to be updated.